tests/test_locality_NeighborList.py

```diff
@@ -15,9 +15,9 @@ def setUp(self):
                                exclude_ii=True)
 
         # Initialize Box and cell list
-        box, points = freud.data.make_random_system(self.L, self.N)
-        self.nq = freud.locality.AABBQuery(box, points)
-        self.nlist = self.nq.query(points,
+        box, self.points = freud.data.make_random_system(self.L, self.N)
```

Comment:
@klarh Was this intentional or an accidental find and replace?

Reply:
Intentional; do you prefer to save the query rather than the points?

Reply:
The self.nq holds its points array. Just use self.nq.points in the test.

Reply:
OK, fixed.